core-bundle/src/Resources/views/Backend/be_undo_label.html.twig

```diff
@@ -0,0 +1,41 @@
+{% trans_default_domain 'contao_default' %}
+
+<div class="tl_undo_header">
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">
+            <span class="date">{{ row.tstamp|date(dateFormat) }}</span>
+            <span class="time">{{ row.tstamp|date(timeFormat) }}</span>
+        </span>
+    </div>
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">{{ 'tl_undo.pid.0'|trans({}, 'contao_tl_undo') }}</span>
+        <strong>{% if user %}{{ user.username }}{% else %}{{ row.pid }}{% endif %}</strong>
+    </div>
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">{{ 'tl_undo.fromTable.0'|trans({}, 'contao_tl_undo') }}</span>
+        <strong>{{ fromTable }}</strong>
+    </div>
+    <div class="tl_undo_header_item">
+        <span class="tl_undo_header_label">ID</span> {{ originalRow.id }}
+    </div>
+    {% if (parent) %}
+        <div class="tl_undo_header_item">
+            <span class="tl_undo_header_label">{{ 'MSC.parent'|trans }}</span>
+            <strong>{{ parent.table }}</strong>
+        </div>
+    {% endif %}
+</div>
+
+<div class="tl_undo_preview">
+    {% if preview is not iterable %}
+        {{ preview|raw }}
+    {% else %}
+        <table style="width: 100%;">
+            <tr>
+            {% for value in preview %}
+                <td>{{ value|raw }}</td>
+            {% endfor %}
+            </tr>
+        </table>
```

Comment:
Shouldn't we use grid/flex here instead of a table?

Reply:
please don't. The backend uses tables everywhere, so this is consistent.

Reply:
The table is used here a) because tables are used every and b) this markup is used, whenever a data container uses `showColumns = true`, so it actually is tabular data (even though just one row). 

I use my own table markup here to not have specific CSS classes, that somebody might have set or used to style the table in the DC list view, e.g. in the member module, each table cell has a class like `col_firstname`, `col_lastname` and so on. There might be custom ones in custom data containers, too. This could potentially break the Undo view, e.g. you set custom widths for table cells.